configure.ac

```diff
@@ -16,14 +16,21 @@ m4_ifdef([AM_SILENT_RULES], [AM_SILENT_RULES([yes])])
 
 PKG_PROG_PKG_CONFIG([0.19])
 
-# Workaround to make aclocal get the right flags
-AC_SUBST(ACLOCAL_AMFLAGS, "\${ACLOCAL_FLAGS}")
+# ****************************
+AM_GNU_GETTEXT([external])
+AM_GNU_GETTEXT_VERSION([0.19.8])
+AM_GNU_GETTEXT_REQUIRE_VERSION([0.19.8])
+
+GETTEXT_PACKAGE=mate-session-wayland
```

Comment:
This needs to be renamed too after renaming ACinit.

Reply:
That came back in when merging your work with mine, fixing it now